ogc/tiles/templates/tiles.go.html

```diff
@@ -1,61 +1,124 @@
+        </tr>
+        <tr>
+          <th>Type</th>
+          <td id="field-type">{{ (index .Config.OgcAPI.Tiles.Types 0) }}</td>
+        </tr>
+        <tr>
+          <th>CRS</th>
+          <td id="field-srs">{{ $defaultSrs.Srs }}</td>
+        </tr>
+        <tr>
+          <th>Metadata Tiles</th>
+          <td id="field-metadata"><a id="href-metadata"
+                                     href="tiles/{{ get $projections $defaultSrs.Srs }}">Link</a>
+          </td>
+        </tr>
+        </tbody>
+      </table>
+    </div>
+    <div class="col-md-8">
+      <app-vectortile-view id="tile-vectortile-view" style="width: 800px; height: 600px;"
+                           tile-url="{{ $baseUrl }}/tiles/{{ get $projections $defaultSrs.Srs }}"
+                           {{ if .Config.OgcAPI.Styles }}style-url="{{ $baseUrl }}/styles/{{ .Config.OgcAPI.Styles.Default }}"{{ end }}
+                           zoom="{{ $defaultSrs.ZoomLevelRange.End }}"
+                           center-x="5.3896944"
+                           center-y="52.1562499"
+                           show-grid>
+      </app-vectortile-view>
+      <table class="table">
+        <tbody>
+        <tr>
+          <th>URL template</th>
+          <td id="field-url-template">{{ $baseUrl }}/tiles/{{ get $projections $defaultSrs.Srs }}/{tileMatrix}/{tileRow}/{tileCol}?f=mvt
+          </td>
+        </tr>
+        <tr>
+          <th>Voorbeeld URL</th>
+          <td id="field-url-example">{{ $baseUrl }}/tiles/{{ get $projections $defaultSrs.Srs }}/{{ $defaultSrs.ZoomLevelRange.End }}/0/0.pbf
+          </td>
+        </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+
+  <script>
+      document.addEventListener('input', function (event) {
+          // only operate on srs dropdown
+          if (event.target.id !== 'srs') return;
+
+          var selectedSrs = event.target.value
+          var srsInteger = selectedSrs.replace(/^EPSG:/, '')
+          var tileset
+          if (selectedSrs === "EPSG:28992") {
+              tileset = "NetherlandsRDNewQuad"
+          } else if (selectedSrs === "EPSG:4258") {
+              tileset = "EuropeanETRS89_GRS80Quad_Draft"
+          } else if (selectedSrs === "EPSG:4258") {
+              tileset = "WebMercatorQuad"
+          }
+
+          var schemaField = document.getElementById('field-schema');
+          var typeField = document.getElementById('field-type');
+          var srsField = document.getElementById('field-srs');
+          var urlTemplateField = document.getElementById('field-url-template');
+          var urlExampleField = document.getElementById('field-url-example');
+          var metadataHref = document.getElementById('href-metadata');
+          var schemaHref = document.getElementById('href-schema');
+          srsField.textContent = selectedSrs;
+          urlTemplateField.textContent = "{{ $baseUrl }}/tiles/" + tileset + "/{tileMatrix}/{tileRow}/{tileCol}?f=mvt";
```

Comment:
Wil je tileMatrix vervangen door `{z}` en `{tileRow}` door `{y}` en `{tileCol}` door `{x}`. Die change heb ik gisteren ook op andere HTML (niet JSON) paginas doorgevoerd zodat het voor gebruikers duidelijker is hoe ze de URL moeten opbouwen n.a.v de tegels in de viewer.

Reply:
ja is afgestemd

Reply:
gefixed